pkgs/tools/misc/hwatch/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub, fetchpatch, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "hwatch";
+  version = "0.3.5";
+
+  src = fetchFromGitHub {
+    owner = "blacknon";
+    repo = pname;
+    rev = "d86479f3b101be7c7b4ed0da3f008b5f2bf583fe";
+    sha256 = "sha256-dGntqLq+wDEsYU1HInX4g7De+WnDZcFpep/zxudlNSk=";
+  };
+
+  cargoPatches = [
```

Comment:
Can you add a comment for future maintainers why the patch is needed?

Reply:
Definitely